src/styles.css

```diff
@@ -115,12 +123,44 @@ a.menu-tab {
   text-decoration: none;
 }
 
-#menu-drawer {
-  height: 100%;
-  overflow: scroll;
+/* WebDesign */
+body {
+  background-color: #e9ddc2;
+}
+
+.buddhanexus {
+  position: absolute;
+  top: -65px;
+  left: 15px;
+  width: 150px;
+  height: 150px;
+}
+
+vaadin-app-layout {
+  /*    background: url("src/assets/img/background_wellcome.jpg"); */
+  background-color: #e9ddc2;
 }
 
-vaadin-drawer-toggle {
-  margin-right: 0;
-  color: #2a3443;
+.menu-tab vaadin-tab {
+  color: #fff;
+}
+
+.head {
+  position: fixed;
+  width: 100%;
+  height: 116px;
+  z-index: 1;
+  text-align: center;
+  background: #a60;
+}
+
+#home {
```

Comment:
this belongs in the styles of the homepage: `src/views/home/home-view.styles.js`